run_kraken.py

```diff
@@ -37,9 +39,39 @@ def main(cfg):
     logging.info("Starting kraken")
 
     # Parse and read the config
-    if os.path.isfile(cfg):
-        with open(cfg, "r") as f:
-            config = yaml.full_load(f)
+    # Initializing flag variable as 0 to test config is a file or not
+    flag = 0
+
+    try:
+        # Checking config is yaml file or not
+        check_yaml_file = urlparse(cfg)
+        path = check_yaml_file.path
+
+        if path[-5:] == ".yaml" or path[-4:] == ".yml":
```

Comment:
@yogananth-subramanian @chaitanyaenr what are your thoughts on this if statement? I asked Sauyma to add in a way to throw an exception if the config file could not be loaded as a yaml.

Reply:
Agreed, we can just raise an exception and exit.

Reply:
+1 for raising an exception and exiting.